HelpSource/Classes/Server.schelp

```diff
@@ -235,6 +235,77 @@ argument:: limit
 The number of times to check for a successful boot. (5 times/sec)
 argument:: onFailure
 A function to evaluate after the server fails to boot. If onFailure is not given, an error message is posted. Providing a function suppresses the error message. If you want to supply a function and print the normal error message, make sure that your function returns "false," e.g. code::s.waitForBoot(onFailure: { ... custom action...; false })::.
+argument:: clock
+The default link::Classes/Clock:: is link::Classes/AppClock::. link::Classes/SystemClock:: and link::Classes/TempoClock:: can be used alternatively.
+discussion::
+code::
+/* example 1a: using the default clock (AppClock) */
+(
+s.waitForBoot {
+	Window.closeAll;
+	fork {
+		600.do { |i|
+			{ w = Window().background_(Color.rand).front }.defer;
+			i = if (i % 4 == 0) { 12 } { 0 }.postln;
+			x = { SinOsc.ar((81 + i).midicps) * Env.perc.ar(Done.freeSelf) * [0.1, -0.1] }.play;
+			1.wait;
+			{ w.close }.defer
+		}
+	}
+}
+)
+::
```

Comment:
Thanks for including examples with this proposed change.

However, I'm not sure what is being illustrated by these examples—it seems that the differences between using different clocks is the goal, but it's not clear at a glance how these examples differ. 
Usually an explanatory phrase in a comment can draw the user's attention to the distinctions you want to highlight.

More clearly defining what should illustrated would also make the example less elaborate. For example, this first example has 4 forked processes (waitForBoot, fork, 2 x defer), and fork doesn't use AppClock.

The details of the example are also not well suited for a quick illustration. This loops 600 times with 1 sec wait times, so the user has to cmd+. to exit out of the example. The example starts by closing all windows, which isn't great if someone already has windows open. This approach might be better served by a short loop, only creating a _single_ window and changing its background color, then using the `onClose` method to kill the process so it just ends when the user closes the window (assuming the loop hasn't already completed).

If you think this still requires 3 separate examples, it's best to create a subsection in the "Examples" section at the end of the help file and link to it, rather than have such a large block of example code in an argument description.